index.html

```diff
@@ -271,6 +271,23 @@ <h1 class="header">I'm Lawrence Glad to see you</h1>
             </div>
 
         </section>
+
+        <section class="contactform">
+            <h1 class="contact-title">Contact me</h1>
+            <p class="contact-description">
+                If you have an application you are interested in developing,
+                a feature that you need built or
+                a project that needs coding. I’d love to help with it
+            </p>
+            <form action="https://formspree.io/f/meqdkrwy" class="contact-form" method="post">
+                <input type="text" class="input-field" name="name" id="name" placeholder="Your Name" maxlength="30"
+                    required>
+                <input type="email" class="input-field" name="email" id="email" placeholder="Email" required>
+                <textarea name="message" class="text-area" id="message" cols="30" rows="10"
+                    placeholder="Write your message here" maxlength="500" required></textarea>
+                <input type="submit" class="submit-btn" value="Get in Touch">
+            </form>
+        </section>
 </body>
```

Comment:
- [ ] Great job working on your portfolio contact form. There is a white space that appears immediately below the contact form which is not supposed to be there according to the Figma design. Kindly ensure that you trim the white space an have the contact form as the last thing to meet all the project requirements. 

**This is how I am seeing the design right now:**

![image](https://user-images.githubusercontent.com/86318284/193842728-b76327c9-2ddf-4029-b9ce-744bed97ea3a.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/193842825-4de485fa-c6b4-482a-8494-0f662aa375c8.png)